### PR TITLE
Correctly enocde the update response when return_body is true

### PR DESCRIPTION
**Pull request by russelldb**

### CODE REVIEW

File: src/riak_kv_pb_crdt.erl [hunk 1]
```diff
@@ -39,9 +39,9 @@
 
 -module(riak_kv_pb_crdt).
 
--include_lib("riak_pb/include/riak_dt_pb.hrl").
--include_lib("riak_pb/include/riak_kv_pb.hrl").
--include_lib("riak_pb/include/riak_pb_kv_codec.hrl").
+-include_lib("../../riak_pb/include/riak_dt_pb.hrl").
```
Comment (seancribbs) [inline src/riak_kv_pb_crdt.erl]: :-1: use rebar deps_dir=..

Comment (russelldb) [inline src/riak_kv_pb_crdt.erl]: Addressed. I've never got to the bottom of why my flymake will not find those includes any other way. I often forget to revert that change. :(

File: src/riak_kv_pb_crdt.erl [hunk 2]
```diff
@@ -340,9 +335,7 @@ upgrade_response(_, {reply, #rpbcountergetresp{value=Val}, State}) ->
 upgrade_response(#rpbcounterupdatereq{}, {reply, #rpbcounterupdateresp{}, State}) ->
     {reply, #dtupdateresp{}, State};
 upgrade_response(_, {reply, #rpbcounterupdateresp{value=Value}, State}) ->
-    {reply, update_resp(riak_kv_crdt:from_mod(?COUNTER_TYPE),
-                        Value,
-                        undefined), State};
+    {reply, #dtupdateresp{counter_value=Value}, State};
```
Comment (seancribbs) [inline src/riak_kv_pb_crdt.erl:338]: Why doesn't this use `encode_update_response`?

Comment (russelldb) [inline src/riak_kv_pb_crdt.erl:338]: No idea, I guess it doesn't need to.

Comment (seancribbs) [inline src/riak_kv_pb_crdt.erl:338]: You're right, sorry my mistake.

**Comment by russelldb** [comment]
Maybe @engelsanchez could have a sneaky peeky?

**Comment by seancribbs** [comment]
Sorry for the latent :+1:. It was the use of `Type` instead of `Mod` that was the major breakage, but it's also nice to cleanup.

**Pull request merged**